### PR TITLE
Fix prettier 

### PR DESCRIPTION
**Pull request by Bachibouzouk**
- add brief description in readme
- impose linting in the CI
- fix package.json

### CODE REVIEW

**Comment by svecon** [comment]
Can we keep the semicolons?

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "prettier --check 'src/server/static/**'"
```
Comment (svecon) [inline package.json]: IMHO "test" command is not meant for linting. Let's add a different command if you want to abstract it. We should also include templates, not just static files.

File: .github/workflows/pythonapp.yml [hunk 2]
```diff
@@ -16,6 +16,19 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cloning the repository
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Prettify the JS Code
+      uses: creyD/prettier_action@v2.0
+      with:
+        prettier_options: 'prettier --check "src/server/static/**"'
```
Comment (svecon) [inline .github/workflows/pythonapp.yml]: Same comment here, what about templates?

File: .github/workflows/pythonapp.yml [hunk 3]
```diff
@@ -16,6 +16,19 @@ jobs:
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Cloning the repository
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - name: Prettify the JS Code
+      uses: creyD/prettier_action@v2.0
+      with:
+        prettier_options: 'prettier --no-semi --check "src/server/static/**"'
```
[the remark this answers is not in the document]
Comment (Bachibouzouk) [reply .github/workflows/pythonapp.yml]: good point

File: package.json (hunk 1 above)
Comment (hnykda) [reply package.json]: Agree, `lint` makes more sense

Comment (Bachibouzouk) [reply package.json]: I tried to define another command but it was not recognized by npm ...

File: .github/workflows/pythonapp.yml (hunk 2 above)
Comment (Bachibouzouk) [reply .github/workflows/pythonapp.yml]: On it

File: package.json (hunk 1 above)
Comment (hnykda) [reply package.json]: OK, didn't know about it, fine merging it as is instead of trying to debug npm

Comment (svecon) [reply package.json]: npm run XXX worked for me

File: .github/workflows/pythonapp.yml (hunk 2 above)
Comment (Bachibouzouk) [reply .github/workflows/pythonapp.yml]: Done in 4383116705921f0c4c3cc63023ca70e6a12fc2c1

File: package.json (hunk 1 above)
Comment (Bachibouzouk) [reply package.json]: Done in 4383116705921f0c4c3cc63023ca70e6a12fc2c1

**Pull request merged**